rir/src/compiler/opt/inline.cpp

```diff
@@ -19,8 +19,8 @@ class TheInliner {
     Closure* function;
     explicit TheInliner(Closure* function) : function(function) {}
 
-    static constexpr size_t MAX_SIZE = 1024;
-    static constexpr size_t MAX_INLINEE_SIZE = 128;
+    static constexpr size_t MAX_SIZE = 10000;
```

Comment:
Just as a comment: I guess more sooner than later it would be quite useful to tweak this values with env vars.